tools/umi_tools/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM continuumio/miniconda3:4.8.2
+LABEL authors="chris.cheshire@crick.ac.uk" \
+      description="Docker image containing all requirements for the nf-core umi_tools module"
+
+# Install apt packages
+RUN apt-get update \
+ && apt-get install -y --no-install-recommends \
+ procps=2:3.3.15-2 \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
```

Comment:
If this command is better than the one in the nf-core base image, it would be nice to propose a PR to update that: https://github.com/nf-core/tools/blob/ebf63095ab222e4df987c4e950ce3839e8784d3e/Dockerfile#L6

Reply:
Its best practice from docker documentation, will raise a PR

Reply:
PR created: https://github.com/nf-core/tools/pull/633